src/App.js

```diff
@@ -41,6 +39,24 @@ function App () {
     }
   }
 
+  const isValidForUpdate = (currentValue, newValue) => {
```

Comment:
**suggestion**: while we're here, what do you think of renaming the parameter `currentValue` to `oldValue`?